package.json

```diff
@@ -20,9 +20,10 @@
     "speculate": "speculate",
     "start": "export $(cat .env | xargs); NODE_ENV=production node build/server.js",
     "storybook": "start-storybook -p 9001 -c .storybook",
-    "test": "npm run test:lint && npm run dataValidate && npm run test:unit",
+    "test": "npm run test:lint && npm run dataValidate && npm run test:unit && npm run test:ampValidate:ci",
     "test:accessibility": "npm run killApp && npm run build && run-p --race start bbcA11y",
-    "test:ampValidate": "npm run killApp && npm run build && run-p --race start amp:validate",
+    "test:ampValidate": "npm run killApp && npm run build -- --hide-modules --colors && run-p --race start amp:validate",
+    "test:ampValidate:ci": "npm run build -- --hide-modules --colors && run-p --race start amp:validate",
```

Comment:
Why do we have two commands that do the same thing? (Apart from one has a `npm run killApp` step)

Feels a bit unnecessary